src/App.js

```diff
@@ -1,14 +1,27 @@
-import React from "react";
+import React, { useState } from "react";
 import AppHeader from "./components/AppHeader";
 import AppContent from "./components/AppContent";
 
 import "./App.sass";
 
+let currentValues = {
+  header: "Header",
+  content: "Content",
+};
+
 function App() {
+  const [appState, setAppState] = useState(currentValues);
```

Comment:
You can set state default value directly here. Otherwise better use spread to copy object.

Reply:
Also, just a semantic thing, but it's not the whole app state that your're storing here. It's card data only. So its name should beter be more meaningful.

Reply:
Also, please check out if app state is actually updated when card is *saved.